src/array.jl

```diff
@@ -960,3 +960,11 @@ function Base.sort!(v::CategoricalVector;
 
     return v
 end
+
+Base.repeat(a::CategoricalArray{T, N, R},
+            counts::Integer...) where {T, N, R} = 
+    CategoricalArray{T, N, R}(repeat(a.refs, counts...), copy(a.pool))
```

Comment:
You know the sources better, but it seems that adding `R` parameter here makes all fail.
What would be the optimal way to call it? (initially I omitted `R` because I thought that it gets inferred from `a.pool`)

Reply:
Ah sorry you're right it's not needed since the pool and refs already carry the information. So you can just remove it.

(I think sometimes `R<:Integer` is needed for dispatch to work, maybe that's the case here?)

Reply:
> (I think sometimes R<:Integer is needed for dispatch to work, maybe that's the case here?)

Adding this did not help, so I removed `R`